test/common/http/conn_manager_impl_fuzz_test.cc

```diff
@@ -123,7 +124,6 @@ class FuzzConfig : public ConnectionManagerConfig {
   MockStreamDecoderFilter* decoder_filter_{};
   MockStreamEncoderFilter* encoder_filter_{};
   NiceMock<MockFilterChainFactory> filter_factory_;
-  absl::optional<std::chrono::milliseconds> idle_timeout_;
```

Comment:
revert change?

Reply:
Uhh, I  did this intentionally to group the timeout functions/vars together. Should I revert?

Reply:
OK that's fine, it wasn't clear to my why it was moved.